File: commands/info.js
const { MessageEmbed } = require('discord.js');

module.exports = {
    name: 'info',
    description: 'Displays information about the bot',
    async execute(message) {
        const embed = new MessageEmbed()
            .setTitle('Bot Information')
            .setDescription('Here is some information about the bot.')
            .setColor('#0099ff');
        
        await message.channel.send({ embeds: [embed] });
    }
};

File: cogs/info.py
import discord
from discord.ext import commands

class Info(commands.Cog):
    def __init__(self, bot: commands.Bot) -> None:
        self.bot = bot

    @commands.command(name="info")
    async def info(self, ctx: commands.Context) -> None:
        embed = discord.Embed(
            title="Bot Information",
            description="Here is some information about the bot.",
            color=discord.Color.blue()
        )

        await ctx.send(embed=embed)

async def setup(bot: commands.Bot) -> None:
    await bot.add_cog(Info(bot))
